tests/test_output.py

```diff
@@ -20,6 +21,8 @@
 from httpie.status import ExitStatus
 from .utils import COLOR, CRLF, HTTP_OK, MockEnvironment, http
 
+JSON_XXSI_PREFIXES = (")]}',\n", ")]}',", 'while(1);', 'for(;;)', ')', ']', '}')
```

Comment:
```suggestion
TEST_JSON_XXSI_PREFIXES = (")]}',\n", ")]}',", 'while(1);', 'for(;;)', ')', ']', '}')
```

(To make it clear these’re the ones we test, no an exhaustiveve list)